src/views/CreateEventView.vue

```diff
@@ -14,11 +17,18 @@ import api from '../api/api';
 export default {
   name: 'CreateEvent',
   components: { EventForm },
+  data: () => ({
+    internalError: true,
+    internalErrorMsg: '',
+  }),
   methods: {
     async createEvent({ event }) {
       const response = await api.createEvent(event);
-      if (response) {
+      if (response.status < 400) {
```

Comment:
Ideally, we shouldn't expect 300 status codes here either. Maybe we should just check for a status in the 200s to indicate success?

Reply:
Yeah good point. I knew 300s aren't for errors but they would still be weird. I'll just check for (200 <= response.status < 300)